docs/troubleshooting.md

```diff
@@ -1,15 +1,43 @@
 # Troubleshooting
 
-## Azure
+## Azure Scenarios
 
 See [CLRIE in Azure](scenarios/azure.md).
 
-## Diagnosing Issues
+## Mitigation
+
+When an issue occurs in production, the first step is to mitigate it so the application continues to run. A common cause of problems are from updates to either CLRIE or to one of the Instrumentation Method client that introduces potential conflicts with another client or incompatiblities with the app.
+
+### Remove CLRIE
+
+The most direct way to stop CLRIE and all Instrumentation Methods is to remove the ICorProfiler environment variables that are being applied to the process (along with a process restart). This may either require updating applicationHost.xdt files, changing the `InstrumentationEngine_EXTENSION_VERSION` Application Setting, or modifying registry keys. Without the ICorProfiler environment variables, the process effectively removes CLRIE & all IMs that are currently running. If the application without CLRIE is still encountering errors, then the issue is something the app owner will need to address.
+
+Alternatively, if the user cannot modify the ICorProfiler environment variable, they may be able to at least remove the dlls which are pointed to by the COR_PROFILER_PATH/CORECLR_PROFILER_PATH environment variables. A process restart will still be required.
```

Comment:
Consider going over the WAD specific mitigation; it's challenging to alter the environment block manually.